issue121_test.go

```diff
@@ -0,0 +1,33 @@
+package mergo
+
+import (
+	"testing"
+)
+
+func TestIssue121WithSliceDeepCopy(t *testing.T) {
+	dst := map[string]interface{}{
+		"inter": map[string]interface{}{
+			"a": "1",
+			"b": "2",
+		},
+	}
+
+	src := map[string]interface{}{
+		"inter": map[string]interface{}{
+			"a": "3",
+			"c": "4",
+		},
+	}
+
+	if err := Merge(&dst, src, WithSliceDeepCopy); err != nil {
+		t.Fatalf("Error during the merge: %v", err)
+	}
+
+	if dst["inter"].(map[string]interface{})["a"].(string) != "3" {
+		t.Error("inter.a should equal '3'")
+	}
+
+	if dst["inter"].(map[string]interface{})["c"].(string) != "4" {
```

Comment:
It would be great also to check `"b"` here

Reply:
@PeerXu Can you check this, please? I would like to merge it.